data/alpha/field/public-body-classifications.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: public-body-classifications
+phase: alpha
+register: public-body-classification
```

Comment:
This is not the key field for the register so is this syntax viable? @michaelabenyohai 
I know what you are trying to do but not sure this is viable